opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -109,7 +110,60 @@ namespace Opm
             adaptiveTimeStepping.reset( new AdaptiveTimeStepping( param_, terminal_output_ ) );
         }
 
-        output_writer_.writeInit();
+
```

Comment:
> or at least the topology (connections) of it.

We *could* - at some stage - create a thin class created by the simulator grid which only carried the information required by the output layer, and then pass that to the output layer; but that could of course easily become the 4.th grid implementation in opm - so not now! 

> So for now I'm willing to include this.

:+1: